web/modules/custom/dpl_example_content/dpl_example_content.module

```diff
@@ -63,4 +63,28 @@ function dpl_example_content_modules_installed(array $modules) : void {
   }
 
   $logger->info("Created {$num_instances} opening hours instances");
+
+  // Set page.front link to /frontpage.
+  $config_site = \Drupal::configFactory()->getEditable('system.site');
+  $current_front_page = $config_site->get('page.front');
+
+  $new_front_page = '/frontpage';
+  if ($current_front_page !== $new_front_page) {
+    $config_site->set('page.front', '/frontpage')->save();
+  }
+
+  $logger->info("Update frontpage link to {$new_front_page}, as part of example content installation.");
```

Comment:
```suggestion
  // Set page.front link to /frontpage.
  $config_site = \Drupal::configFactory()->getEditable('system.site');
  $new_front_page = '/frontpage';
  $config_site->set('page.front', $new_front_page)->save();
  $logger->info("Update frontpage link to {$new_front_page}, as part of example content installation.");
```

Personally I do not see the need to check the value of the frontpage setting. The logged statement seems to assume that the frontpage value will be set regardless.